### PR TITLE
818 dashboard modify input data section for multiple examples

### LINKED ISSUE #818: Dashboard: modifu input data section for multiple examples
instead of “load example data” make selection boxes in the “input data” section for each specific example

image

### PR DESCRIPTION
**Pull request by laurasootes**
This PR a specifications for each example in the dashboard in the left menu (fixes #818).
Additionally, upon example selection, text will be shown explaining what the example is (adds to #816 and #819)

Please check if the explanatory text is sufficient.

### CODE REVIEW

File: dianna/dashboard/pages/Time_series.py [hunk 1]
```diff
@@ -20,26 +20,35 @@
...
     ts_label_file = (label_directory / 'weather_data_labels.txt')
 
+    st.markdown(
+        """This example demonstrates the use of DIANNA 
+        on a pre-trained binary classification model for season prediction.
+        The input data is the
+        [weather prediction dataset](https://zenodo.org/records/5071376).
+        This classification model uses time (days) as function of mean temperature to predict if the whole time series is either summer or winter.
+        Using a chosen XAI method the relevance scores are displayed on top of the timeseries. The days contributing positively towards the classification decision are indicated in red and those who contribute negatively in blue.
+        """)
```
Comment (SarahAlidoost) [inline dianna/dashboard/pages/Time_series.py]:
```suggestion
    st.markdown(
        """This example demonstrates the use of DIANNA
        on a pre-trained binary classification model for season prediction. The
        input data is the [weather prediction
        dataset](https://zenodo.org/records/5071376). This classification model
        uses time (days) as function of mean temperature to predict if the whole
        time series is either summer or winter. Using a chosen XAI method the
        relevance scores are displayed on top of the timeseries. The days
        contributing positively towards the classification decision are
        indicated in red and those who contribute negatively in blue.
        """)
```

Comment (SarahAlidoost) [reply dianna/dashboard/pages/Time_series.py]: fixing ruff error on `Line too long`

File: dianna/dashboard/pages/Text.py [hunk 2]
```diff
@@ -18,27 +18,33 @@
...
-if load_example:
+if load_example_moviesentiment:
     text_input = 'The movie started out great but the ending was dissappointing'
     text_model_file = model_directory / 'movie_review_model.onnx'
     text_label_file = label_directory / 'labels_text.txt'
 
+    st.markdown(
+        """
+        This example demonstrates the use of DIANNA on the
+        [Stanford Sentiment Treebank dataset](https://nlp.stanford.edu/sentiment/index.html) which contains one-sentence movie reviews. A pre-trained neural network classifier is used, which identifies whether a movie review is positive or negative.
```
Comment (SarahAlidoost) [inline dianna/dashboard/pages/Text.py]:
```suggestion
        """
        This example demonstrates the use of DIANNA on the [Stanford Sentiment
        Treebank dataset](https://nlp.stanford.edu/sentiment/index.html) which
        contains one-sentence movie reviews. A pre-trained neural network
        classifier is used, which identifies whether a movie review is positive
        or negative.
```

Comment (SarahAlidoost) [reply dianna/dashboard/pages/Text.py]: fixing ruff error on `Line too long`

File: dianna/dashboard/pages/Images.py [hunk 3]
```diff
@@ -19,29 +19,37 @@
...
     image_file = (data_directory / 'digit0.jpg')
     image_model_file = (model_directory / 'mnist_model_tf.onnx')
     image_label_file = (label_directory / 'labels_mnist.txt')
 
+    st.markdown(
+        """
+        This example demonstrates the use of DIANNA on a pretrained binary [MNIST](https://yann.lecun.com/exdb/mnist/) model using a hand-written digit images.
+        The model predict for an image of a hand-written 0 or 1, which of the two it most likely is.
+        This example visualizes the relevance attributions for each pixel/super-pixel by displaying them on top of the input image.
+        """
```
Comment (SarahAlidoost) [inline dianna/dashboard/pages/Images.py]:
```suggestion
        """
        This example demonstrates the use of DIANNA on a pretrained binary
        [MNIST](https://yann.lecun.com/exdb/mnist/) model using a hand-written
        digit images. The model predict for an image of a hand-written 0 or 1,
        which of the two it most likely is. This example visualizes the
        relevance attributions for each pixel/super-pixel by displaying them on
        top of the input image.
        """
```

Comment (SarahAlidoost) [reply dianna/dashboard/pages/Images.py]: fixing ruff error on `Line too long`

**Review by SarahAlidoost — CHANGES_REQUESTED** [review]
@laurasootes thanks for adding clear explanations for each example :1st_place_medal: . Some text used in `test_dashboard.py` should be changed accordingly, for example [this one](https://github.com/dianna-ai/dianna/blob/430be09d51c26cfc5fc7ed9b87126cb7965a544b/tests/test_dashboard.py#L101). 

Also, I noticed a title is set for each page as `st.title('Image explanation')` or `st.title('Time series explanation')`, .... Can we make them more informative? :thinking: 

I fixed some `line too long` errors, but let's check if there are other linter errors. Can you please run `ruff check` on the changed files?

**Comment by laurasootes** [comment]
hi @SarahAlidoost, thanks for your review. I updated the dashboard tests accordingly and checked the line lengths, after your fixes they are all good now. 

I agree with your comment about the page title. I have added it to issue #816 where in which all tetx-related things will be updated

**Review by SarahAlidoost — APPROVED** [review]
@laurasootes thanks for addressing the comments. Changes look good :+1:  Before merging, can you please change the base branch of this pull request from `817-dashboard-move-menu` to `main`. Because #830 is already merged. Also, only after changing the base to `main`, github action will run automatically on this pull request.

**Pull request merged**